<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu_2</class>
 <widget class="QDialog" name="UserMenu_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_data_godzina">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wyloguj">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyloguj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Wstecz">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Wstecz</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_BazaSamochodow">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Wypozycz">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>751</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Wypożycz Samochód</string>
   </property>
   <widget class="QPushButton" name="pushButton_Sprawdz">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprawdź</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BLIK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Karta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Blik">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BLIK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Karta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Karta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Zaplac_Wypozycz">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapłać i wypożycz</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>227</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Podaj ID Samochodu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_PodajID"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>174</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Na ile dni ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_IleDni"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>129</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Cena">
       <property name="text">
        <string>Cena</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_KoncowaData">
       <property name="text">
        <string>Końcowa data wynajmu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;Baza Samochodów&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
